src/main/cliapp/src/components/SingleReferenceAdditionalFieldData.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export function SingleReferenceAdditionalFieldData({ fieldData }) {
+	if (fieldData) {
+		if (fieldData["curie"] !== '' && (fieldData['cross_references'] && fieldData['cross_references'].length > 0) ){
+			for(let i=0; i<fieldData['cross_references'].length; i++){
+				if(fieldData['cross_references'][i].curie.indexOf('PMID:') > -1){
+					return <div className="p-info">{fieldData['cross_references'][i].curie}</div>;
+				}else if(fieldData['cross_references'][i].curie.indexOf('FB:') > -1){
+					return <div className="p-info">{fieldData['cross_references'][i].curie}</div>;
+				}else if(fieldData['cross_references'][i].curie.indexOf('MGI:') > -1){
+					return <div className="p-info">{fieldData['cross_references'][i].curie}</div>;
+				}else if(fieldData['cross_references'][i].curie.indexOf('RGD:') > -1){
+					return <div className="p-info">{fieldData['cross_references'][i].curie}</div>;
+				}else if(fieldData['cross_references'][i].curie.indexOf('SGD:') > -1){
+					return <div className="p-info">{fieldData['cross_references'][i].curie}</div>;
+				}else if(fieldData['cross_references'][i].curie.indexOf('WB:') > -1){
+					return <div className="p-info">{fieldData['cross_references'][i].curie}</div>;
+				}else if(fieldData['cross_references'][i].curie.indexOf('ZFIN:') > -1){
+					return <div className="p-info">{fieldData['cross_references'][i].curie}</div>;
```

Comment:
@kthorat-prog are these not all the same?

Reply:
no ....
also, I am not pushing this branch. I have created a new one and working on it. This branch seems to have a lot of changes which are already on alpha. To avoid any merge complications, I will push changes on new branch.

Reply:
It just wasn't updated

Reply:
Maybe I am missing something here but are not all these cases doing the same thing?

Reply:
Couldn't we just split the `fieldData['cross_references'][i].curie` via `:` and then check that the prefix is in a list.

Reply:
No. All cases have different stuff to display.

Reply:
But the return is the same